DogeNetWebClient/src/app/store/reducers/user-info.reducer.ts

```diff
@@ -0,0 +1,36 @@
+import { Action, createReducer, on } from '@ngrx/store';
+import * as Actions from 'src/app/store/actions/user-info.actions';
+import { AccountDetailsModel } from 'src/app/core/api/models';
+import { UserState } from '../states/UserState';
+
+const initialState: UserState = {
```

Comment:
Could we move the state into the separate file as it would be used separatly?

Reply:
one file for state interface one for reducer and one for initial state?

Reply:
or could I just export initial state from state file?